File: assignment/average.cpp
#include<iostream>
using namespace std;
class Average
{
    int num1,num2,num3,num4,num5,Average;
    public :void avg()
    {
        cout<<"Enter the first number: \n";
        cin>>num1;
        cout<<"Enter the second number: \n";
        cin>>num2;
        cout<<"Enter the third number: \n";
        cin>>num3;
        cout<<"Enter the fourth number: \n";
        cin>>num4;
        cout<<"Enter the fifth number:\n";
        cin>>num5;
        Average=(num1+num2+num3+num4+num5)/5;
        cout<<"The average of the above five numbers is: "<<Average;
    }
};
int main()
{
    Average obj;
    obj.avg();
}

File: Java Assignment 1/Average.java
import java.util.*;
class Average
{
int num1,num2,num3,num4,num5,avg;
void accept()
{
Scanner sc=new Scanner(System.in);
System.out.println("Enter five numbers: ");
num1=sc.nextInt();
num2=sc.nextInt();
num3=sc.nextInt();
num4=sc.nextInt();
num5=sc.nextInt();
}
void display()
{
avg=((num1+num2+num3+num4+num5)/5);
System.out.println("The average of five numbers is: "+avg);
}
public static void main(String...args)
{
Average obj=new Average();
obj.accept();
obj.display();
}
}